File: content/python/templates/file-io.py
# File Input/Output

# Write to file
with open("output.txt", "w") as f:
    f.write("Hello, World!\n")
    f.write("This is a test.\n")

# Read from file
with open("output.txt", "r") as f:
    content = f.read()
    print(content)

# Read line by line
with open("output.txt", "r") as f:
    for line in f:
        print(line.strip())

File: content/cpp/templates/file-io.cpp
// File Input/Output

#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main() {
    // Write to file
    ofstream outFile("output.txt");
    if (outFile.is_open()) {
        outFile << "Hello, World!" << endl;
        outFile << "This is a test." << endl;
        outFile.close();
    }
    
    // Read from file
    ifstream inFile("output.txt");
    string line;
    
    if (inFile.is_open()) {
        cout << "File contents:" << endl;
        while (getline(inFile, line)) {
            cout << line << endl;
        }
        inFile.close();
    }
    
    return 0;
}
